zerver/tests/test_messages.py

```diff
@@ -1924,6 +1924,26 @@ def do_schedule_message(self, msg_type: str, to: str, msg: str,
         result = self.client_post("/json/messages", payload)
         return result
 
+    def test_handle_deferred_message_by_user_ids(self) -> None:
+        content = "Test message"
+        defer_until = timezone_now().replace(tzinfo=None) + datetime.timedelta(days=1)
+        defer_until_str = str(defer_until)
+
+        result = handle_deferred_message(
+            sender=self.example_user('hamlet'),
+            client=get_client('test'),
+            message_type_name='private',
+            message_to=[self.example_user('othello').id],
+            topic_name=None,
+            message_content=content,
+            delivery_type='remind',
+            defer_until=defer_until_str,
+            tz_guess='',
+            forwarder_user_profile=self.example_user('hamlet'),
+            realm=None
+        )
+        self.assert_json_success(result)
```

Comment:
I don't understand this commit; can you explain why you're adding this brittle test?

Reply:
Oops that's my bad, this commit message is supposed to say `mypy: Accept user IDs in handle_deferred_message.` (which is what the commit is called locally on my branch so I just forgot to push probably) and is supposed to prevent mypy errors because callers may pass in integer user IDs to these functions now.

The brittle test shouldn't be there either. Initially, I had thought that I would not add support for user IDs to these methods and only focus on `send_message_backend` first. So I went down the rabbit hole of doing a manual type cast inside `handle_deferred_message`, and that's when I added the test. Anyway, updating now! :)